neon_homeassistant_skill/__init__.py

```diff
@@ -5,17 +5,22 @@
 from ovos_workshop.decorators import intent_handler
 from ovos_workshop.skills import OVOSSkill
 
+
 def chunks(lst, n_len) -> List[list]:
     """Split list into n-length chunks."""
     n_len = max(1, n_len)
     return [lst[i : i + n_len] for i in range(0, len(lst), n_len)]
 
 
+DEFAULT_SETTINGS: dict = {
+    "connected": True,
```

Comment:
Adding this to `settingsmeta.yml` should populate the default setting during init

Reply:
Worth doing down the line, I think...for now I'll leave it explicit